bytestring.cabal

```diff
@@ -55,7 +55,8 @@ Maintainer:          Duncan Coutts <duncan@community.haskell.org>
 Homepage:            https://github.com/haskell/bytestring
 Bug-reports:         https://github.com/haskell/bytestring/issues
 Tested-With:         GHC==8.10.1, GHC==8.8.1, GHC==8.6.3, GHC==8.4.4, GHC==8.2.2,
```

Comment:
Could we please test against GHC 8.6.5 (instead of 8.6.3) and GHC 8.8.3 (instead of 8.8.1)?

Reply:
seconded.

Reply:
I'd double check whether 8.8.3 in the ppa repo.  I thought I had not seen it there when I last looked, specifically looking to make the suggested change.  I agree it is best done.

Reply:
I was wrong, 8.6.5 and 8.8.3 are both available, will do
https://launchpad.net/~hvr/+archive/ubuntu/ghc/+index?batch=75&memo=164&start=164